src/microbe_stage/editor/CellEditorComponent.cs

```diff
@@ -1861,7 +1861,7 @@ private void RenderHighlightedOrganelle(int q, int r, int rotation, OrganelleDef
         bool showModel = !hadDuplicate;
 
         // Model
-        if (showModel && shownOrganelle.TryGetGraphicsScene(out var modelInfo))
+        if (showModel && shownOrganelle.TryGetGraphicsScene(null, out var modelInfo))
```

Comment:
When moving an organelle, shouldn't this method take in a parameter specifying the organelle upgrades so that the visuals are consistent?

Reply:
Upgrades are available from PlacedOrganelle and there is not much of it available for this line (`shownOrganelle` is OrganelleDefinition)

Reply:
When moving an organelle, the variable `MovingPlacedHex` is of type `OrganelleTemplate` which contains upgrade data. That's why I made that comment. So updating this method to allow passing in upgrade data would hopefully make it relatively simple to pass those upgrades in when moving a hex.